code/drasil-data/Data/Drasil/Constraints.hs

```diff
@@ -3,5 +3,5 @@ module Data.Drasil.Constraints where
 import Language.Drasil
 
 gtZeroConstr, probConstr :: Constraint
-gtZeroConstr = physc $ UpFrom  (Exc, 0)
-probConstr   = physc $ Bounded (Inc, 0) (Inc, 1)
+gtZeroConstr = physc $ UpFrom  (Exc, Int 0)
```

Comment:
As this is a "physical constraint", it is really about reals, so this is the wrong type.

Reply:
This should be fixed now, thank you